src/test/datascience/jupyter/kernels/installationPrompts.vscode.test.ts

```diff
@@ -148,7 +150,6 @@ suite('DataScience Install IPyKernel (slow) (install)', function () {
                 disposables
             );
             const installed = createDeferred();
-            const interpreterPath = which ? venvPythonPath : venvNoRegPath;
```

Comment:
Looks like the Python extension code has changed, we're getting a response even before the installation has completed.
In the past we'd wait for the installation to complete (even when running the installer in the terminal).
Based on the logs we can see that ipykernel is indeed getting installed (files are getting created as part of ipykernel installation).

@rchiodo When you bring in the changes for package installation we can just enable this test (as we'll no longer be using the terminal).

Hence disabling the test for now